src/Core/Domain/Header/Asset/Asset.php

```diff
@@ -103,7 +103,8 @@ private function getFilePath(): string
         $realPath = realpath($path);
         if ($realPath === false || !str_starts_with($realPath, $rootRealPath)) {
             throw new InvalidArgumentException(
-                'Did you run webpack already? Otherwise the file does not exist or is in a location that is not allowed: ' . $path
+                'Did you run webpack already?
+                Otherwise the file does not exist or is in a location that is not allowed: ' . $path
```

Comment:
Moet je hier niet concatten ipv newline. Want nu gaat die newline ook in de output staan.